Governance/publish/publish-procedure.md

```diff
@@ -0,0 +1,97 @@
+````
+
+````{tab-item} Metadata
+
+**Abstract**
+
+This ...
+
+
+**Self-describing files**
+Most of the file formats commonly used in climate are self-describing. An example are the attributes in a netCDF file. It is important to use these as much as possible to give a precise and correct definition of the data itself following available conventions. Using the self-describing properties of a file has two important advantages: it keeps important information with the data itself, and these files attributes are used by discipline software to simplify data analysis.
+Publishers that deal regularly with climate data usually require at least for the [CF](../concepts/cf-conventions) and/or [ACDD](../concepts/acdd-conventions) conventions to be followed, however it is always worth applying them when possible even if they are not required for publication.
+
+```{note}
+Please note we cover CF conventions application and potential issues when they are used incorrectly in the [technical pages](../tech/conventions).
+```
+
+**Auxiliary files**
+
+These could be any kind of text, tabular files or other formats like markup (xml, html, json, md etc.) that add information on the data. These could also be actual data files, for example the ancillary files used to run a model simulation. 
+````
+
+````{tab-item} Formatting files
+
+**Files organisation**
+The way files are organised in folders, their names and sizes should consider both how the files will be distributed and how they will be used. For example, the protocol used to download the files might have a maximum size allowed, likewise having to list a lot of small files it's inefficient when loading an html page. Names also should be descriptive enough that a file can be recognised easily after has been downloaded as being part of a specific dataset. We covered [files organisation and naming](../tech/drs-names) in the technical pages of this book, however, it is important to check the publisher instructions in this regard, or, if none are available online, contacting them about it as early as possible in the publishing process.  
+
+**Conventions**
+It is important to use [conventions](../concepts/conventions) and [controlled vocabularies](../concepts/controlled-vocab) whenever possible, both official ones, like CF conventions for file attributes, and others which are not a requirement but have become common practice in the climate community (e.g. CMIP variable names). As some of these conventions also apply to folder and file names, it is important to be consistent and use the same terms in the files, names and descriptions. 
```

Comment:
You could mention the CORDEX convention as an example, too. One thing I learned recently is that the CORDEX convention keeps changing (aka updating), and if one is not aware of the changing convention happening every couple of weeks, one might miss updating the data file and not be CORDEX aligned anymore. Currently, there are data description and meta-data differences between ACS-CCAM and QLD-CCAM published CMIP6 CORDEX data because of that.

Reply:
I was trying to give some generic advice, hence I mentioned only CF, we should have a publishing with ESGF where all these quirks can be added. CMOR, CORDEX etc. I will copy this comment to the relevant issue. And also could be added to other-conventions file? We should be covering cortex there

Reply:
see issue #28